applications/ecallmgr/src/ecallmgr_originate.erl

```diff
@@ -768,6 +768,8 @@ maybe_send_originate_uuid({'fs', UUID}, Pid, #state{server_id=ServerId
                                                    }) ->
     CtlQ = gen_listener:queue_name(Pid),
     publish_originate_uuid(ServerId, UUID, JObj, CtlQ);
+maybe_send_originate_uuid({'api', UUID}, Pid, State) ->
+    maybe_send_originate_uuid({'fs', UUID}, Pid, State);
```

Comment:
/only/ delivered if this patch is applied. That L770 publish_originate_uuid doesn't happen if this function is called with {'api', UUID} in the original code. You simply do not get the control queue.